src/components/contextual/pages/pool/MyPoolBalancesCard.vue

```diff
@@ -106,6 +118,20 @@ function fiatLabelFor(index: number, address: string): string {
   const fiatValue = toFiat(propTokenAmounts.value[index], address);
   return fNum(fiatValue, currency.value);
 }
+
+function navigateToPoolMigration(pool: FullPool) {
+  router.push({
+    name: 'migrate-pool',
+    params: {
+      from: pool.id,
+      to: POOLS.IdsMap[networkId.value]?.bbAaveUSD
```

Comment:
Should `bbAaveUSD` be in a variable so this supports future boosted pools? Could have it in a config map of normal => boosted pool equivalents.